.test_durations

```diff
@@ -0,0 +1,1816 @@
+{
```

Comment:
is this file required?

Reply:
It was coming from `pytest-split` library, but I removed it now until we will have a clear way to use it (since we need to maintain up to date information about how much time each test takes).